examples/nextjs-together-llamaindex/frontend/src/app/components/Examples.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import { useState } from "react";
+import { Example, examples } from "./examplesList";
+import { triggerWorkflow } from "@/app/actions/trigger";
+
+const Examples = () => {
+  const [loading, setLoading] = useState(false);
+  const [output, setOutput] = useState({});
+
+  const handleButtonClick = async (example: Example) => {
+    setLoading(true);
+    setOutput({});
+    try {
+      const result = await triggerWorkflow(example.workflowName, example.input);
```

Comment:
Hello, it's Nomenclature Police👮‍♀️
You have violated RNO p12 3a where workflows are _scheduled_, not _triggered_.
Please comply to protocol else you can be arrested.

On duty, your Nomenclature Police 🚓

Reply:
Yes this is from existing NextJs example, will fix.